docs/docker.md

```diff
@@ -1,134 +1,151 @@
 # Running an epoch node on Docker
 
 This document describes:
+* [How to join the Roma network using Docker](#roma-network);
+* [How to use the Docker image](#docker-image);
 * [How to join the testnet using Docker](#testnet);
 * [How to run a local network using Docker](#localnet).
 
-You must have [docker installed](https://docs.docker.com/engine/installation/) on a host machine.
+You must have [docker installed](https://docs.docker.com/engine/installation/) on a host machine and **be familiar [how to operate docker containers](https://docs.docker.com)**.
 
-## Testnet
-
-### Docker Image
+## Roma Network
 
-Docker image is automatically build and published on [DockerHub](https://hub.docker.com/r/aeternity/epoch/). All tagged source code versions have their own docker image tag as well. Latest tagged ("stable") version is tagged with `latest`.
-Master branch of the source code is tagged as `dev`.
+The default node configuration is sufficient to join the Roma network:
 
-To pull the latest "stable" image run:
 ```bash
 docker pull aeternity/epoch
+docker run -P aeternity/epoch
```

Comment:
Please clarify impact of publishing all exposed ports. I am considering user running Docker host with public IP. (And listening address.)

Reply:
I used explicit ports in a07970b2